### PR TITLE
Get methods add to rest interface

### PR DESCRIPTION
**Pull request by phmp**

### CODE REVIEW

File: src/main/java/main/RestManager.java [hunk 1]
```diff
@@ -3,10 +3,9 @@
 import controller.OfferManager;
 import model.Offer;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.RequestBody;
```
Comment (LordVader14) [inline src/main/java/main/RestManager.java]: I suggest to leave these imports as they were. We are new to Spring so we want to know which Spring classes we are using exactly.

File: src/main/java/main/RestManager.java [hunk 2]
```diff
@@ -3,10 +3,9 @@
...
-import org.springframework.web.bind.annotation.RequestMapping;
```
Comment (LordVader14) [inline src/main/java/main/RestManager.java]: See my above comment

File: src/main/java/main/RestManager.java [hunk 3]
```diff
@@ -3,10 +3,9 @@
...
-import org.springframework.web.bind.annotation.RestController;
```
Comment (LordVader14) [inline src/main/java/main/RestManager.java]: See my above comment

File: src/main/java/model/Offer.java [hunk 4]
```diff
@@ -5,7 +5,7 @@
  */
 public class Offer {
 
-    private long id;
+    private Long id;
```
Comment (LordVader14) [inline src/main/java/model/Offer.java]: Do we really need Long here? Just to consider. Do you think we will ever reach 2,147,483,647 orders? I hope so :) but it seems to be unreal for now.

File: src/main/java/main/RestManager.java (hunk 1 above)
Comment (phmp) [reply src/main/java/main/RestManager.java]: ok, you are right

File: src/main/java/model/Offer.java (hunk 4 above)
Comment (phmp) [reply src/main/java/model/Offer.java]: Why not? I think that Long is better here. @gaabral what do you think?

Comment (gaabral) [reply src/main/java/model/Offer.java]: I would leave Long as it is.

Comment (phmp) [reply src/main/java/model/Offer.java]: Any option here would be good. So would you approve Long type? Approve is required to merge branch to master.

**Review by gaabral — APPROVED** [review]

**Pull request merged**